wolfssl/wolfcrypt/types.h

```diff
@@ -46,6 +46,7 @@
 	    #endif
 	    typedef unsigned short word16;
 	    typedef unsigned int   word32;
+	    typedef unsigned char  word24[3];
```

Comment:
Shouldn't word24 be based on byte like before?

Reply:
I figured its better to map to the source of the typedef, but I guess using `byte` allows for use of it in the case where byte is a define. I'll switch it up.